app/celery/letters_pdf_tasks.py

```diff
@@ -159,6 +168,28 @@ def collate_letter_pdfs_for_day(date=None):
         )
 
 
+def get_key_and_size_of_letters_to_be_sent_to_print(print_run_date):
+    letters_awaiting_sending = dao_get_letters_to_be_printed(print_run_date)
+
+    letter_pdfs = []
+    for letter in letters_awaiting_sending:
+        letter_file_name = get_letter_pdf_filename(
```

Comment:
I won't lie, I copied and adapted this code from elsewhere and then matched my tests results to what I was getting which feels naughty. I'm wondering if I should introduce something like `get_letter_pdf_filename_from_notification` which can then be unit tested and make sure that given a letter, the correct arguments are passed into the `get_letter_pdf_filename` function?

Reply:
Decided not to make any changes as not to go down a rabbit hole, plus I understood the code much better after you explained what `dont_use_sending_date` was for so I'm more confident in my code changes.